package.json

```diff
@@ -30,6 +30,7 @@
     "test": "ava test/test.js"
   },
   "dependencies": {
+    "@types/node": "^10.11.7",
```

Comment:
I'm not too keen on adding this as a dependency to the project. Is there another way to accomplish the appropriate typing without this dependency?